.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,85 @@
+---
+name: trivy
+
+on:
+  workflow_call:
```

Comment:
There's a lot of inputs you do not need here, or at least you are not using.

If you want to work similar to the workflow `docker-build` I believe you should only use:
* name
* namespace
* tag
* registry

You can remove all the others, including `image-ref` as it can be rebuilt from the param above.

Then you should set as secret parameters:
* REGISTRY_LOGIN
* REGISTRY_PASSWORD

So under workflow_call:
```yaml
  workflow_call:
    inputs:
    ... # the inputs we mentioned
    secrets:
      REGISTRY_LOGIN:
        required: false
      REGISTRY_PASSWORD:
        required: false
```

Reply:
Done ! Just, I did not use `name` in Trivy's Workflow